tests/test_state_generators.py

```diff
@@ -14,3 +18,35 @@ def test__generate_segmentation_layer_default_values():
 
     assert "pick" in state
     assert state["pick"] is False
+
+
+def test__generate_configuration_default_values():
+    state = combine_json_layers(layers=[{"type": "image", "volumeRendering": "OK", "name": "myname"}], scale=1.0)
+
+    assert "enableLayerColorWidget" in state
```

Comment:
In this case I think we'll need to pull out the color widget related changes, because I don't think we can merge this PR until horizontal stacking and our scroll fix are available. Unlike some of the other changes which can sit in the state and wait until neuroglancer understands the state changes